meta-lmp-bsp/conf/machine/include/lmp-machine-custom.inc

```diff
@@ -496,6 +496,7 @@ UBOOT_MACHINE:imx8mn-lpddr4-evk = "imx8mn_evk_config"
 BOOTSCR_LOAD_ADDR:imx8mn-lpddr4-evk = "0x44800000"
 MACHINE_EXTRA_RRECOMMENDS:append:imx8mn-lpddr4-evk = " bluetooth-attach"
 IMXBOOT_TARGETS:imx8mn-lpddr4-evk ?= "flash_evk_spl"
+MACHINE_FEATURES:append:imx8mn-lpddr4-evk = " mxm-mwifiex-load"
```

Comment:
Do we also need a similar change for imx8mn-ddr4-evk?

Reply:
@ricardosalveti I didn't have a chance to look at the imx8mn-ddr4 doc. It might use another wi-fi module.
@mwasilew have you seen the same issue on imx8mn-ddr4?

Reply:
I'll check tomorrow. The board is currently broken in the LAB - so I don't know.

Reply:
DDR4 uses CYW43455 (brcmfmac I guess) and LPDDR4 88W8987, so we should be fine with this change here only.